source/voice_control/android.markdown

```diff
@@ -76,9 +84,9 @@ Assist is available on Wear OS using the [Home Assistant Companion App](https://
 
 The way how Assist can be set up on your phone may differ depending on your version of Wear OS.
 
-1. After [installing the companion app](https://companion.home-assistant.io/docs/getting_started/) and connecting it to your Home Assistant, Assist might appear automatically on the **Apps screen**.
-2. If it does not, visit your watch companion app or the Wear OS app and select **Add tile** inside the **Tiles** area.
-3. Tap the **Assist** tile to add it to your watch:
+1. After [installing the companion app](https://companion.home-assistant.io/docs/getting_started/) on your watch and connecting it to your Home Assistant, Assist might appear automatically on the **Apps screen**.
+   - If it does not, visit your watch companion app or the Wear OS app and select **Add tile** inside the **Tiles** area.
+2. Tap the **Assist** tile to add it to your watch:
```

Comment:
These steps don't make a lot of sense to me? You start out by talking about the Assist icon/app in the apps list. It should _always_ appear here, just like the main app icon should always appear.

If you cannot find the icon, you suggests adding a tile which is unrelated to apps/icons in the apps list.